### PR TITLE
Feature: hide tokens

### PR DESCRIPTION
**Pull request by Abyscuit**
### Feature

* \[x] Added entry to ./changelog
* \[ ] Is this a user submitted feature request?
  * \[ ] Link to issue:
* \[ ] Add test cases or procedure
* \[x] Add PR label

### CODE REVIEW

**Review by hsurf22 — APPROVED** [review]

File: src/modules/balance/components/TokenDeleteCustom.vue [hunk 1]
```diff
@@ -146,15 +185,66 @@ export default {
...
   },
   methods: {
-    ...mapActions('custom', ['deleteAll', 'deleteToken']),
+    ...mapActions('custom', [
+      'deleteAll',
+      'deleteToken',
+      'setHiddenToken',
+      'deleteHiddenToken'
+    ]),
+    // preselectTokens() {
```
Comment (gamalielhere) [inline src/modules/balance/components/TokenDeleteCustom.vue:227]: remove commented code

File: src/modules/balance/components/TokenDeleteCustom.vue [hunk 2]
```diff
@@ -173,6 +263,13 @@ export default {
       newObj.tokenImg = item.img ? item.img : this.network.type.icon;
       return newObj;
     },
+    /**
+     * @returns formatted values to display correctly on token table
+     */
+    // formatHiddenValues(item) {
```
Comment (gamalielhere) [inline src/modules/balance/components/TokenDeleteCustom.vue:269]: remove commented code

File: src/modules/balance/components/TokenDeleteCustom.vue (hunk 1 above)
Comment (Abyscuit) [reply src/modules/balance/components/TokenDeleteCustom.vue]: The file is no longer used as I replaced it with the edit token form so I will delete it

File: src/components/MewTokenContainer/MewTokenContainer.vue [hunk 3]
```diff
@@ -0,0 +1,156 @@
+<template>
```
Comment (gamalielhere) [inline src/components/MewTokenContainer/MewTokenContainer.vue:1]: Delete file and move changes to mew components.

**Review by gamalielhere — CHANGES_REQUESTED** [review]

**Review by gamalielhere — APPROVED** [review]

**Review by andrewdahn — APPROVED** [review]

**Pull request merged**